stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -346,7 +360,8 @@ public class MessageInputView : ConstraintLayout {
             val hasValidContent = hasContent && !messageLengthExceeded
 
             attachmentsButton.isVisible = style.attachButtonEnabled && !commandMode
-            commandsButton.isVisible = style.lightningButtonEnabled && !commandMode && suggestionListViewAttached
+            commandsButton.isVisible =
```

Comment:
If the list of commands returned by backend is null, it should be hidden as well. Right?

Reply:
Nice catch! Thanks